docs/src/pages/guides/interoperability.md

```diff
@@ -257,3 +257,7 @@ function GlobalCSSButton() {
 
 export default GlobalCSSButton;
 ```
+
+[![Edit Button](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/2zv5m0j37p)
+
+**Note:** JSS inject their styles at the bottom of the `<head />`. If you don't want to mark style attributes with **!important**, you need to change [the CSS injection order](/customization/css-in-js#css-injection-order) as we do in the demo.
```

Comment:
'JSS injects its styles'

Reply:
x4 the same mistake 😬 .

Reply:
Do you want to commit a quick fix directly on v1-beta? :)

Reply:
I'll submit a separate PR.

Reply:
#YOLO directly on HEAD is good some time :).

Reply:
I made a few other changes that will need your review and approval. 😄